.github/workflows/documentation.yml

```diff
@@ -0,0 +1,56 @@
+name: Build Documentation
+
+on:
+  push:
+    branches:
+      - 'main'
```

Comment:
This workflow targets `main`, but there is not such a branch - Perhaps Master needs renaming as main?

Reply:
Thanks, I've opened #321 for this.

Reply:
@yaswant , thanks, fixed in 41576067353cd0cf3447b86ca5b478de8d706935